.github/workflows/testing.yml

```diff
@@ -6,7 +6,7 @@ on:
   workflow_dispatch:
```

Comment:
Nit: I would replace `/ru/` in the link in the commit message (and PR description) with `/en/` to make it readable for everyone.

Reply:
Sure, fixed.